src/app/routes/gateway.js

```diff
@@ -94,4 +94,23 @@ module.exports = (Router, Service) => {
         res.status(500).send({ error: err.message });
       });
   });
+
+  Router.get('/gateway/registerCompleted', basicAuth, (req, res) => {
```

Comment:
Could you use `/gateway/users/?email=...` as is more semantic and respects [REST best practices](https://restfulapi.net/resource-naming/)?